### ISSUE TITLE
Donut WLDP and AMSI patching enabled

### ISSUE THREAD

**Issue by CodeXTF2** [BishopFox/sliver#1759]
**Describe the bug**
When generating shellcode in Sliver, Donut has the AMSI and WLDP options enabled, which can add detections.

**To Reproduce**
1. Start Sliver
2. generate --http {any ip here} --os windows --format shellcode

**Expected behavior**
Elastic flags the beacon for patching AMSI and WLDP even though -e (which would normally enable AMSI patching) is disabled.

OS Windows (implant)

**Comment by rkervella** [comment]
The `--evasion` flag has no impact on this. Right now, most of the Donut loader options are hardcoded: https://github.com/BishopFox/sliver/blob/7676fc6c37fc33ea17a6ce34784bec1e8107dce6/server/generate/donut.go#L50

I should probably update that so it can be user controlled.

**Comment by CodeXTF2** [comment]
yeah im aware. I just referenced the --evasion flag in terms of intended behaviour. For now manually using donut with -b1 on sliver binaries works fine

**Comment by moloch--** [comment]
You can now use `generate` flags to control this.
